packages/venia-ui/lib/queries/getCustomer.graphql

```diff
@@ -0,0 +1,10 @@
+query getCustomer {
+    # This would ideally be a fragment that could be shared where needed.
+    # Please keep shape in sync with createAccount.graphql until this is supported.
+    customer {
+        id
+        email
+        firstname
+        lastname
```

Comment:
Is this all the necessary data we were using before? I know we tend to spread the entire REST response into the store so I definitely want to make sure we are reducing that "noise".

Reply:
Only usage of this state I found was in `useMyAccount`, `const { email, firstname, lastname } = currentUser;`, and `useAuthModal` and that damn `id`; so this should cover it. Agree we should only fetch what we need.

Reply:
I do anticipate this will expand, to pre-fill billing and shipping details, but that functionality does not currently exist.